lib/livebook/zta.ex

```diff
@@ -57,4 +57,10 @@ defmodule Livebook.ZTA do
   def put(name, value) do
     :ets.insert(__MODULE__, [{name, value}])
   end
+
+  def provider_name(nil), do: "None"
```

Comment:
This should be in Livebook.Config because this module will be refactored out , but it is fine to be as is :)